app/views/homes/_about_us.html.erb

```diff
@@ -10,7 +10,7 @@
           the month.
         </p>
         <div class="about-us__cta">
-          <%= link_to("Join us on Slack!", "#") %>
+          <%= link_to("Stay informed", "https://www.meetup.com/bostonrb/", target: "_blank") %>
```

Comment:
Just an alternative, what do you think of "Join the Community", or "Join our Community"?

Reply:
Sure!